programs/merkle-distributor/src/lib.rs

```diff
@@ -294,6 +375,23 @@ pub struct MerkleDistributor {
     pub num_nodes_claimed: u64,
 }
 
+#[account]
+#[derive(Default)]
+pub struct Config {
```

Comment:
did we initiate merkle distributor with extra space s.t. we can add these directly to the distributor?

Reply:
Yeah, init_if_need space 256 when UpdateConfig